Modules/IO/MeshBase/test/itkMeshFileReaderWriterTest.cxx

```diff
@@ -49,8 +51,21 @@ itkMeshFileReaderWriterTest(int argc, char * argv[])
   reader->SetFileName(inputFileName);
   ITK_TRY_EXPECT_EXCEPTION(reader->Update());
 
+  inputFileName = "NonExistingFile.vtk";
+  reader->SetFileName(inputFileName);
+  ITK_TRY_EXPECT_EXCEPTION(reader->Update());
+
+  inputFileName = "UnsupportedExtensionFile.mesh";
+  std::ofstream unsupportedExtFileOfs(inputFileName);
+  reader->SetFileName(inputFileName);
+  ITK_TRY_EXPECT_EXCEPTION(reader->Update());
+
+  // Delete the created file
+  std::remove(inputFileName.c_str());
```

Comment:
I've realized that the file is not being effectively removed, on the one hand because it is lacking the path, and on the other hand because when doing PR #3383, I've realized that even when the path is added and is correct, the file is still not removed from the file system.

Linked to https://github.com/InsightSoftwareConsortium/ITK/pull/3383#discussion_r851647569, and the rest of the comments in that PR.

@dzenanz @tbirdso @Leengit any suggestion to fix this? Thanks.